### PR TITLE
alert to install curl/git

### PR DESCRIPTION
**Pull request by 0xilis**
pretty self explanatory, tested on Ubuntu :P.

### CODE REVIEW

**Comment by junepark678** [comment]
why do you need curl or git?

**Review by Fofo989 — APPROVED** [review]

**Review by Yeet1000 — COMMENTED** [review]
I'd suggest you don't mention APT since some new Linux users on other distros might try to use the apt command expecting it to work

**Review by junepark678 — APPROVED** [review]

**Comment by junepark678** [comment]
> I'd suggest you don't mention APT since some new Linux users on other distros might try to use the apt command expecting it to work

they wouldn't be compiling palera1n if they don't know

File: download_deps.sh [hunk 1]
```diff
@@ -1,3 +1,15 @@
+if ! command -v curl &> /dev/null
```
Comment (Vendicated) [inline download_deps.sh:1]: &> is a bashism, so you should either remove it (just > works fine) or add a shebang
```suggestion
if ! command -v curl > /dev/null
```

**Review by jacksonsmitg207 — APPROVED** [review]

File: download_deps.sh (hunk 1 above)
Comment (0xilis) [reply download_deps.sh]: ty

**Comment by 0xilis** [comment]
it's been some months and there have been no updates on getting this PR'd... commenting as a reminder :P.

**Comment by 0xilis** [comment]
UPDATE: seems that download_deps.sh was removed from this repo, closing now :'(.

**Pull request closed without merging**